package.json

```diff
@@ -2,13 +2,14 @@
   "name": "@openmrs/root-config",
   "version": "1.0.0",
   "description": "A javascript module for setting up and configuring your SPA ",
-  "main": "dist/root-config-lib.js",
+  "main": "dist/root-config.js",
```

Comment:
The main file will be the one that registers all the applications automatically. For those who want to configure which applications are registered, they will import root-config-lib manually. See related https://github.com/AMPATH/ampath-esm-root-config/pull/1/files#diff-ed9242f7605faeb4b2cb4294644c0188R2